home-screen-widget/profiles/README.md

```diff
@@ -27,9 +27,27 @@ Now, we can start the development server:
 yarn serve
 ```
 
-In order to use your local widget in a tenant, it is required to make it accessible from the outside world. One way is to use [NGROK](https://ngrok.com/) to expose your local development. Next, you will need to make a request to the API to register the widget type and grant access for your tenant.
+To use your widget in a test tenant, you will need to expose your development server to testing device. If both devices are in the same network, this should be simple. You should see the address of the development server on the console. Otherwise, we recommend to use a solution like [NGROK](https://ngrok.com/). You can start a simple forward with ``ngrok http 8080``. Next, you will need to make a request to the API to register the widget type and grant access for your tenant.
 
-## Required Steps for Widget Development
+## Registering with API
+
+If you followed above steps, you should have a local development server running on a public accessible URL e.g. https://2af62c7779d5.ngrok.io. Navigating to https://2af62c7779d5.ngrok.io/home-screen-widget-profiles/js/profiles.js in your browser should show you the content of the whole bundle.
+
+Next, you will have to register your new widget type with the Beekeeper API and configure your Home Screen to include your new widget. For both steps, you can use the provided Python 3 script in the ``scripts`` folder. 
+
+To create a new widget type:
+
+```shell
+python3 scripts/configure_home_screen.py --tenantUrl https://<tenant_url> --token <access_token> --op add_widget_type --widgetType <widget_type> --widgetUrl <widget_url>
```

Comment:
Should we reference how to get an access token? Also that it needs to be global admin? 

Ref 1: https://developers.beekeeper.io/v2/welcome/authentication
or Ref 2: https://adminhelp.beekeeper.io/hc/en-us/articles/360002574420-Creating-Bots